packages/core-database-postgres/lib/spv.js

```diff
@@ -24,7 +24,7 @@ module.exports = class SPV {
    * @return {void}
    */
   async build (height) {
-    this.activeDelegates = config.getConstants(height).activeDelegates
+    this.maxDelegates = config.getConstants(height).activeDelegates
```

Comment:
maybe we should change the naming in the config as well

Reply:
Brian is working on this afaik.